style.css

```diff
@@ -0,0 +1,1086 @@
+* {
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+}
+
+/* from Carlos */
+
+.mobile-nav {
+  display: none;
+}
+
+nav a {
+  text-decoration: none;
+  color: #555;
+  font-family: Maquette, Helvetica Neue, Arial, sans-serif;
+}
+
+.top-nav {
+  width: 100%;
+  display: inline-flex;
+  background-color: #131313;
+  justify-content: space-between;
+  height: 30px;
+  align-items: center;
+}
+
+.topNav-left {
+  display: flex;
+  align-self: flex-end;
+  height: 100%;
+  align-items: center;
+  justify-content: space-between;
+  width: 30%;
+  margin-left: 20px;
+}
+
+.topNav-right {
+  display: flex;
+  align-self: flex-end;
+  height: 100%;
+  align-items: center;
+  justify-content: space-between;
+  width: 15%;
+  margin-right: 20px;
+}
+
+.main-nav {
+  width: 100%;
+  display: inline-flex;
+  justify-content: space-between;
+  height: 48px;
+  margin: 0;
+  top: 0;
+}
+
+.main-nav a {
+  color: #aaa;
+}
+
+.nav-logo {
+  width: auto;
+  height: 100%;
+  display: none;
+}
+
+.mainNav-left {
+  align-items: center;
+  width: 47%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.mainNav-right {
+  display: flex;
+  justify-content: center;
+  align-self: flex-end;
+  align-items: center;
+  margin-right: 20px;
+  height: 100%;
+  width: auto;
+}
+
+.mainNav-right i {
+  font-size: 1.5em;
+  color: #aaa;
+}
+
+.header-img {
+  width: 99vw;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  height: 30vw;
+}
+
+.header-img .main-header-img {
+  width: 60%;
+  height: 100%;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-clip: content-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  position: relative;
+}
+
+.header-img .sub-header-img {
+  width: 39%;
+  height: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: column;
+}
+
+.sub-img-text > div {
+  z-index: 10;
+}
+
+.header-img .sub-header-img > div {
+  width: 100%;
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ]  Please add a class to `<div>` element` and delete the left unnecessary rules

**Example:**
```CSS
/* before */
.header-img .sub-header-img > div {

/* after */
.divtag-class {
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_